packages/node/src/indexer/store.service.ts

```diff
@@ -767,6 +768,37 @@ group by
           throw new Error(`Failed to bulkCreate Entity ${entity}: ${e}`);
         }
       },
+
+      bulkUpdate: async (
+        entity: string,
+        data: Entity[],
+        fields?: string[],
+      ): Promise<void> => {
+        try {
+          const model = this.sequelize.model(entity);
+          const modelFields =
```

Comment:
I think this needs to be below the `assert`

Reply:
> Did we want to throw an error if `bulkUpdate` is called with historical for now?

Sure, will have a follow up pr for historical